util/stop/stopper_test.go

```diff
@@ -15,13 +15,19 @@
 //
 // Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-package util
+package stop
 
 import (
 	"testing"
 	"time"
+
+	"github.com/cockroachdb/cockroach/testutils/fakeflags"
```

Comment:
I don't think it's bad enough to go out to env variables just yet. As it is it's easy to cover with CI (and even if we don't do that, someone will notice when they need it). I've moved the tests to `stop_test`. The only case where we really need `fakeflags` is that of `util/encoding` (since `util/log` imports that). Refactoring `util/log` so that the proto parsing stuff is a separate package would untie that too.